--- pacman.conf	2022-05-11 11:18:28.390075791 +0300
+++ pacman.conf.new	2022-05-11 11:18:54.433410914 +0300
@@ -30,10 +30,10 @@
 
 # Misc options
 #UseSyslog
-#Color
+Color
 #NoProgressBar
 CheckSpace
-#VerbosePkgLists
+VerbosePkgLists
 #ParallelDownloads = 5
 
 # By default, pacman accepts packages signed by keys that its local keyring
@@ -90,8 +90,8 @@
 #[multilib-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
