--- extension.js	2022-02-17 15:00:23.731316908 +0300
+++ extension.js.new	2022-02-17 15:00:35.851379461 +0300
@@ -387,6 +387,10 @@
             initially decorated? ${win._maximusDecoratedOriginal}`
     );
 
+    if (!shouldAffect(win)) {
+        return;
+    }
+
     if (settings.undecorateAll) {
         setDecorated(win, false);
     } else if (useAutoUndecorList && autoUndecorAppsRegexp.test(win.get_wm_class_instance())) {
